package.json

```diff
@@ -103,6 +103,7 @@
     "@angularclass/hmr": "1.2.0",
     "@angularclass/hmr-loader": "3.0.2",
     "angular-in-memory-web-api": "0.1.13",
+    "angular2-datatable": "^0.5.3",
```

Comment:
Please don't use the ^ here. Things break when they update their library and you run install and don't realized they have upgraded. These can be used in some places but it should be intentional. 

As a production system we need to control what versions we are running because that is what we tested.